src/tests/Common/GenerateHWIntrinsicTests/GenerateHWIntrinsicTests_Arm.cs

```diff
@@ -232,20 +232,10 @@
                 {
                     for (var i = 1; i < RetElementCount; i++)
                     {
-                        var iterResult = (mask[i] != 0) ? 0 : falseVal[i];
-                        if (mask[i] != 0)
+                        if ((mask[i] != 0) ? ({ValidateRemainingResults}) : (falseVal[i] != result[i]))
```

Comment:
can we not have the ternary conditional operator inside if-check? for simplicity, please have them separate.

Reply:
Done